### PR TITLE
Extraction of modules from main cloud

### PR DESCRIPTION
**Pull request by TheMeinerLP**
This pull request, removes some modules(CloudPerms, NPC Selector, SignSystem, SimpleNameTags, CloudFlare, NotifySystem) from main cloud. The idea to rebuild the modules in external reposetories and use later teh rest api of updateserver with a install command to install the modules.

Please feel free to review and say what can i improve. I try my best to clean up some code

### CODE REVIEW

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/CloudConfig.java [hunk 1]
```diff
@@ -44,8 +44,7 @@
         Paths.get("local", "templates"),
         Paths.get("local", "plugins"),
         Paths.get("local", "cache"),
-        Paths.get("groups"),
-        Paths.get("modules")
+        Paths.get("groups")
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/CloudConfig.java:47]: You can leave in the "modules" directory, even if it is left empty.

File: cloudnet-core/src/main/resources/modules/modules.properties [hunk 2]
```diff
@@ -1,8 +1,3 @@
 #
 # Copyright (c) Tarek Hosni El Alaoui 2017
-#
-CloudNet-Service-PermissionModule=1.0.0.2
-CloudNet-Service-MobModule=1.0.0.2
-CloudNet-Service-SignsModule=1.0.0.2
-CloudNet-Service-CloudFlareModule=1.0.0.3
-CloudNet-Service-NotifyServiceModule=1.0.0.2
\ No newline at end of file
+#
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/resources/modules/modules.properties]: You may delete this file

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/CloudConfig.java (hunk 1 above)
Comment (TheMeinerLP) [reply cloudnet-core/src/main/java/de/dytanic/cloudnetcore/CloudConfig.java:47]: Look in the module manager, there are becomes to create the folder

File: cloudnet-core/src/main/resources/modules/modules.properties (hunk 2 above)
Comment (TheMeinerLP) [reply cloudnet-core/src/main/resources/modules/modules.properties]: I deleted the file

**Comment by GiantTreeLP** [comment]
Will be merged once all modules are in development branches of their separate repositories.

**Comment by GiantTreeLP** [comment]
> Will be merged once all modules are in development branches of their separate repositories.

Push

**Pull request merged**